### DISCUSSION TITLE
How to integreate `cordova-plugin-firebase-messaging`  ?

### DISCUSSION THREAD

**Discussion by realtebo** [quasarframework/quasar#13896] [CLI - Capacitor / Cordova modes]
We succesfully integrate google firebase cloud messaging (FCM) into our quasar 2 app. It works both in foreground and in background.

Now we need to add it as cordova plugin to enable this feature in android version.

I see instructions here: https://github.com/chemerisuk/cordova-plugin-firebase-messaging 

But .. where must I add this code?  where have I access to 'cordova' object in the quasar app?

**Comment by MilosPaunovic** [comment]
[https://quasar.dev/quasar-cli-vite/developing-cordova-apps/cordova-plugins](https://quasar.dev/quasar-cli-vite/developing-cordova-apps/cordova-plugins)
